cmake/pcl_targets.cmake

```
@@ -114,7 +114,7 @@ macro(PCL_SUBSYS_DEPEND _var _name)
     if(SUBSYS_EXT_DEPS)
       foreach(_dep ${SUBSYS_EXT_DEPS})
         string(TOUPPER "${_dep}_found" EXT_DEP_FOUND)
-        if(NOT ${EXT_DEP_FOUND} AND (NOT (${EXT_DEP_FOUND} STREQUAL "TRUE")))
+        if(NOT ${EXT_DEP_FOUND})
```

```suggestion
if(NOT EXT_DEP_FOUND)
```

No need to explicitly apply variable expansion inside `if` statements.